code_search/kubeflow/vendor/kubeflow/examples@defc235463799d5600001ee0ed6ef68f7af24a17/prototypes/tf-batch-predict.jsonnet

```diff
@@ -0,0 +1,143 @@
+// @apiVersion 0.1
```

Comment:
should this be in different PR?

Reply:
I'm not following? You want one PR for adding vendor and one for adding TB?

Reply:
The TensorBoard prototoype comes from the examples package. So we need to install the entire examples package in order to get TB which brings in other prototypes as well.

Reply:
ok, makes sense, didn't notice we add whole component